assets/js/blocks/price-filter/types.ts

```diff
@@ -1,8 +1,12 @@
-export interface Attributes {
+interface Attributes {
```

Comment:
Changing `export interface Attributes {...}` to `interface Attributes {...}` led to a TS error in https://github.com/woocommerce/woocommerce-blocks/blob/a6856d203924e50cad9e3fbcf492911b0313a761/assets/js/blocks/price-filter/edit.tsx#L28

Reply:
Yes I just noticed, both block.tsx and edit.tsx are using the props differently, I'll revert the export change and push.